fpdcc/document-search
# Add a download button for PDFs

## Overview

Add a button to the detail page allowing users to download PDFs.

This change is intended to help users with older browsers better navigate PDFs.

## Testing Instructions

* Start your dev environment
* Log into BrowserStack and open up a browser with Windows 10 and Edge 17
* Navigate to http://localhost:8000/licenses/1/
* Scroll to the bottom of the page and confirm that the embedded PDF doesn't have PDF controls (due to a bug in older versions of Edge)
* Scroll back to the top and click the "Download" button
* Confirm the PDF gets opened in a window where controls are visible
